docs/application/native/guides/error/error-handling.md

```diff
@@ -30,7 +30,7 @@ How to query error codes for a certain API function is determined by how the fun
 
   ```
   Remarks:
-  The specific error code can be obtained using the get_last_result() function. Error codes are described in Exception section.
+  The specific error code can be obtained using the get_last_result() function in [Common Error](../../api/common/latest/group__CAPI__COMMON__ERROR.html). Error codes are described in Exception section.
   To see all of the available error values, please refer to the Exceptions section in each API function description.
   For example:
```

Comment:
Since the " ``` " symbol is involved, I cannot properly provide code suggestions.

Please move the " ``` " in Line 31 to Line 35, just before the "For example:"

Otherwise it appears like this. 

![image](https://github.com/user-attachments/assets/044711e7-7f72-4422-963c-734b046c7153)